composer.json

```diff
@@ -199,6 +199,9 @@
       "select2": {
         "url": "https://github.com/colemanw/select2/archive/v3.5-civicrm-1.0.zip"
       },
+      "js-yaml": {
+        "url": "https://github.com/nodeca/js-yaml/archive/3.13.1.zip"
+      },
```

Comment:
@totten I think you did this slightly differently in your gist (which I can't seem to find now). Notably you added some `ignore` items. What were they again?

Reply:
Yup, that's right. I'll push them up momentarily.